packages/Notification/Notification.jsx

```diff
@@ -30,14 +28,6 @@ const renderIcon = icon => <DecorativeIcon symbol={icon.symbol} variant={icon.co
 const renderContent = (variant, children) => {
```

Comment:
The `content` variable, and actually the entire `renderContent` function, is not necessary anymore. I'd recommend removing it and inlining this into the main component.